src/theme/HawaTheme.js

```diff
@@ -6,7 +6,8 @@ export const CreateHawaTheme = (
   primaryActionColor,
   primaryLayoutColor,
   primaryActionTextColor,
-  mainFont
+  mainFont,
+  breakpointsValues
```

Comment:
I might need ur opinion about this, basically I letting people define theme's breakpoint